reporting_app/util.py

```diff
@@ -33,10 +34,106 @@ def tab_set_cfg(title, tables):
         'tables': tables
     }
 
+def get_token():
+    return 'Token ' + encode_string(request.cookies.get('remember_token'))
+
 
 def capitalise(word):
     return word[0].upper() + word[1:]
 
 
 def snake_case(text):
     return text.lower().replace(' ', '_')
+
+
+
+
+def chart_variables(endpoint, data):
+
```

Comment:
Is this function used anywhere? I can see it being imported in `reporting_app.__init__`, but no usages.